README.md

```diff
@@ -33,3 +33,8 @@ This is a component that animates through an image sprite.
 - startFrame **{number}** - the frame to start animation
 - timeout **{number}** - the amount of millisecond between frame
 - stopLastFrame **{bool}** - stops animation from looping
+
+**Only required for two-dimensional sprites**
+
+- frameCount **{number}** - the total frame count of the sprite
```

Comment:
Thanks! Well, since we are able to either read in a horizontal or vertical direction, `wrapAfter` describes the process more generally because `columns` would apply only to reading horizontally.

```
frameCount: 7, wrapAfter: 4, direction: "horizontal"
1  2  3  4
5  6  7
```
vs.
```
frameCount: 7, wrapAfter: 4, direction: "vertical"
1  5
2  6
3  7
4
```